### DISCUSSION TITLE
TIF to PDF conversion results in high brightness

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#1182] [Help]
I have noticed that converting TIF files to PDF results in a high brightness-washed outlook. See the example here
[test.zip](https://github.com/dlemstra/Magick.NET/files/8638505/test.zip)

The code I use is below. Any idea why this happens?

```
 using (var collection = new MagickImageCollection(Conversion.SourceFile.ToString()))
            {                
                using (var fs = File.Create($@"{Conversion.DestinationLocation}\result.pdf"))
                {
                    collection.Write(fs, MagickFormat.Pdf);
                }
            }
```

![image](https://user-images.githubusercontent.com/2582291/167092987-65a84a32-09ae-479a-9a88-2b8a229bb724.png)

**Comment by dlemstra** [comment]
I don't understand why this happens but it looks like this has to do with how the PDF is rendered. When I convert the PDF back to an image I am getting the correct output. Maybe you could ask for more information here: https://github.com/ImageMagick/ImageMagick/discussions?
